src/main/java/uk/gov/companieshouse/web/accounts/transformer/smallfull/BalanceSheetTransformer.java

```diff
@@ -1,4 +1,15 @@
 package uk.gov.companieshouse.web.accounts.transformer.smallfull;
 
+import uk.gov.companieshouse.web.accounts.model.smallfull.BalanceSheet;
+import uk.gov.companieshouse.web.accounts.model.smallfull.CalledUpShareCapitalNotPaid;
+
 public class BalanceSheetTransformer {
+
+    public BalanceSheet getBalanceSheet() {
```

Comment:
Is this method set in stone? Or is it purely just to get something in? I only ask because in the api we have overloaded transform methods that you can call to transform data between objects.

Reply:
We may rename this method, though the problem we're facing is that when converting data to send to the API we'll be taking the balance sheet and converting it to a current period and to a previous period. The method names must be clear and concise to avoid confusion, though the given method name is subject to change.

Reply:
Are you not using any mapping frameworks as we discussed before? Additionally, will you be converting objects from Web to API and vice versa from the top level parent? If so, you should be able to just use bean copy utils

Reply:
We may well end up using `BeanUtils#copyProperties`, though this will be decided on a case by case basis depending on how much the web and api models differ.

Reply:
I'm just trying to understand this more but regarding your comment about 
>we'll be taking the balance sheet and converting it to a current period and to a previous period.

If you are to send data to the API, surely you would be sending the current period with its balance sheet data and not the balance sheet itself. Additionally, should we be converting balance sheet to period? I would assume its balanceSheet to balanceSheet?

Reply:
I'll retrieve my comment for now as I feel there is an understanding I'm probably missing in the web to api mapping.